### PR TITLE
Add Fedora 34 to CI

### PR DESCRIPTION
**Pull request by aminvakil**
##### SUMMARY
See https://github.com/ansible-collections/overview/issues/45#issuecomment-830220740

##### ISSUE TYPE
- CI Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

**Comment by aminvakil** [comment]
It seems that it has not been pushed to quay yet, will return to this a couple of days later.

File: .azure-pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -289,7 +289,7 @@ stages:
           targets:
             - name: CentOS 8
               test: centos8
-            - name: Fedora 32
+            - name: Fedora 33
```
Comment (felixfontein) [inline .azure-pipelines/azure-pipelines.yml:292]: Thanks for spotting that!

**Comment by felixfontein** [comment]
Looks like there's a problem with postgresql setup under Fedora 34. Maybe we should wait until community.postgresql applies this change and then see what they changed ;-) (CC @Andersson007)

**Comment by aminvakil** [comment]
> Looks like there's a problem with postgresql setup under Fedora 34. Maybe we should wait until community.postgresql applies this change and then see what they changed ;-) (CC @Andersson007)

Right! Let's wait...

**Comment by aminvakil** [comment]
@felixfontein Next step after https://github.com/ansible-collections/community.general/pull/2384/commits/ff0c899a8650e78967a1933b93fd8015695a6a61 would be disabling the entire test on Fedora 34 IMHO, would you like me to do that?

**Comment by felixfontein** [comment]
Let's keep this for now. Can you create an issue for tracking this?

**Pull request merged**